packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -32,3 +32,7 @@ settings-subscriptions = Subscriptions
 ## legal footer
 terms = Terms of Service
 privacy = Privacy Notice
+
+
```

Comment:
For consistency we could get a comment here ## plan details

Reply:
totally, thanks for catching that